Makefile

```diff
@@ -0,0 +1,48 @@
+GO_VET_CMD      = go tool vet --all -shadow
+GO_DIRS         = $(shell ls -d */ | \grep -v vendor)
+GO_PKGS         = $(shell go list ./... | \grep -v vendor)
+GO_SRCS         = $(shell find . -type f -name '*.go')
+
+# assets removes the existing bindata.go and regenerates it.
+.PHONY          : assets
```

Comment:
nitpick... There should be no space between `.PHONY` and `: assets`.

Reply:
I tend to keep the target and deps aligned, but if you prefer, I can remove it.

Reply:
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
I suggest you read this link and format the Makefile. :) I prefer we follow the Makefile convention.

Reply:
I am quite familiar with the make docs ;) However, even in that link, they do not have a consistent style, some time using spaces some time not. But no worries, I'll push the change and remove the spaces.

Reply:
Thanks. :)